### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by shlomoklahr**
## Checklist

- [x] Only one project/change is in this pull request
- [x] Addition in alphabetical order
- [x] Appropriate language icon(s) added if applicable
- [x] Has a commit from less than 2 years ago
- [x] Has a **clear** README in English

### CODE REVIEW

**Comment by shlomoklahr** [comment]
Formatting for PodcastMenu addition somehow got messed up. I've fixed it

**Comment by serhii-londar** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please update the Pull Request title to contain the library name</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: README.md [hunk 1]
```diff
@@ -439,8 +439,7 @@ Mac app wrapper around Google's stand-alone Android Messenger. ![javascript_icon
 
 - [Cumulonimbus](https://github.com/z-------------/CPod) - Simple, beautiful podcast app. ![javascript_icon]
 - [mkchromecast](https://github.com/muammar/mkchromecast) - Cast macOS and Linux Audio/Video to your Google Cast and Sonos Devices. ![python_icon]
-- [Overcast](https://overcast.fm/) on your Mac's menu bar so you can listen to your favorite podcasts while you work. ![swift_icon]
-- [PodcastMenu](https://github.com/insidegui/PodcastMenu) - PodcastMenu is a simple app which puts. ![swift_icon]
+- [PodcastMenu](https://github.com/insidegui/PodcastMenu) - PodcastMenu is a simple app which puts [Overcast](https://overcast.fm/) on your Mac's menu bar so you can listen to your favorite podcasts while you work. ![swift_icon]
```
Comment (serhii-londar) [inline README.md]: Why do you put two apps in one line?

**Comment by serhii-londar** [comment]
@shlomoklahr Thank you! 👍

**Pull request merged**